packages/sitecore-jss-angular/src/components/generic-link.directive.spec.ts

```diff
@@ -171,7 +171,9 @@ describe('<a *scGenericLink />', () => {
 @Component({
   selector: 'test-router-link-children',
   template: `
-    <a *scGenericLink="field; editable: editable; attrs: attrs; extras: extras" id="my-link"><span *ngIf="true">hello world</span></a>
+    <a *scGenericLink="field; editable: editable; attrs: attrs; extras: extras" id="my-link"
```

Comment:
@ambrauer I'm not sure that it makes sense to add some rules, disable length, because we don't have `templates` inside `sitecore-jss-angular`, we have directives, and inside tests some `templates`

Reply:
@sc-illiakovalenko I agree, if it's just limited to tests this is fine as-is.